components/device-mgt/org.wso2.carbon.device.mgt.api/src/test/java/org/wso2/carbon/device/mgt/jaxrs/service/impl/DeviceTypeManagementAdminServiceTest.java

```diff
@@ -0,0 +1,252 @@
+import org.testng.annotations.Test;
+import org.wso2.carbon.device.mgt.common.DeviceManagementException;
+import org.wso2.carbon.device.mgt.common.spi.DeviceTypeGeneratorService;
+import org.wso2.carbon.device.mgt.core.dto.DeviceType;
+import org.wso2.carbon.device.mgt.core.service.DeviceManagementProviderService;
+import org.wso2.carbon.device.mgt.core.service.DeviceManagementProviderServiceImpl;
+import org.wso2.carbon.device.mgt.extensions.device.type.template.DeviceTypeGeneratorServiceImpl;
+import org.wso2.carbon.device.mgt.jaxrs.service.api.admin.DeviceTypeManagementAdminService;
+import org.wso2.carbon.device.mgt.jaxrs.service.impl.admin.DeviceTypeManagementAdminServiceImpl;
+import org.wso2.carbon.device.mgt.jaxrs.service.impl.util.DeviceMgtAPITestHelper;
+import org.wso2.carbon.device.mgt.jaxrs.util.DeviceMgtAPIUtils;
+
+import javax.ws.rs.core.Response;
+
+import static org.mockito.MockitoAnnotations.initMocks;
+
+/**
+ * This class holds the unit tests for the class {@link DeviceTypeManagementAdminService}
+ */
+@PowerMockIgnore("javax.ws.rs.*")
+@SuppressStaticInitializationFor({"org.wso2.carbon.device.mgt.jaxrs.util.DeviceMgtAPIUtils"})
+@PrepareForTest({DeviceMgtAPIUtils.class, DeviceManagementProviderService.class})
+public class DeviceTypeManagementAdminServiceTest {
+
+    private static final Log log = LogFactory.getLog(DeviceTypeManagementAdminService.class);
+    private static final String TEST_DEVICE_TYPE = "TEST-DEVICE-TYPE";
+    private static final String TEST_DEVICE_TYPE_1 = "DUMMY-DEVICE-TYPE-1";
+    private static final String TEST_DEVICE_TYPE_2 = "DUMMY DEVICE TYPE";
+    private static final int TEST_DEVICE_TYPE_ID = 12345;
+    private static final int TEST_DEVICE_TYPE_ID_1 = 123452;
+    private static final int TEST_DEVICE_TYPE_ID_2 = 121233452;
+    private DeviceTypeManagementAdminService deviceTypeManagementAdminService;
+    private DeviceManagementProviderService deviceManagementProviderService;
+    private DeviceTypeGeneratorService deviceTypeGeneratorService;
+
+    @ObjectFactory
+    public IObjectFactory getObjectFactory() {
+        return new org.powermock.modules.testng.PowerMockObjectFactory();
+    }
+
+    @BeforeClass
+    public void init() throws DeviceManagementException {
+        log.info("Initializing DeviceTypeManagementAdmin tests");
+        initMocks(this);
+        this.deviceManagementProviderService = Mockito
+                .mock(DeviceManagementProviderServiceImpl.class, Mockito.RETURNS_MOCKS);
+        this.deviceTypeGeneratorService = Mockito.mock(DeviceTypeGeneratorServiceImpl.class, Mockito.RETURNS_MOCKS);
+        this.deviceTypeManagementAdminService = new DeviceTypeManagementAdminServiceImpl();
+    }
+
+    @Test(description = "Test get all the device types.")
+    public void testGetDeviceTypes() {
+        PowerMockito.stub(PowerMockito.method(DeviceMgtAPIUtils.class, "getDeviceManagementService"))
+                .toReturn(this.deviceManagementProviderService);
+
```

Comment:
Remove unwanted extra lines

Reply:
This is fixed in commit 
https://github.com/wso2/carbon-device-mgt/pull/1082/commits/57975b2e58d4756d58ddb255efad220dd3b3558e